### PR TITLE
Create account high level API

### PR DESCRIPTION
**Pull request by petervanderkadecent**

### CODE REVIEW

File: src/api/AccountApi.ts [hunk 1]
```diff
@@ -262,4 +263,26 @@ export class AccountApi extends BaseApi {
...
+     * @param registrar registrar account id
+     * @param name name of new account
+     * @param publicKey publicKey for new account
+     * @param fee {@link AssetAmount} fee for the operation, if left undefined the fee will be computed in DCT asset
+     *
+     * @return a transaction confirmation
+     */
+    public createNew(
+        credentials: Credentials,
+        registrar: ChainObject,
```
Comment (ShockiTV) [inline src/api/AccountApi.ts]: is not registrar already provided as credentials.account ?

Comment (petervanderkadecent) [reply src/api/AccountApi.ts]: Fixed

File: src/api/AccountApi.ts [hunk 2]
```diff
@@ -262,4 +263,23 @@ export class AccountApi extends BaseApi {
...
+     * Creates new account.
+     *
+     * @param credentials registrar account credentials
+     * @param name name of new account
+     * @param publicKey publicKey for new account
+     * @param fee {@link AssetAmount} fee for the operation, if left undefined the fee will be computed in DCT asset
+     *
+     * @return a transaction confirmation
+     */
+    public createNew(
```
Comment (fl3xman) [inline src/api/AccountApi.ts]: Just call it `create` as in other sdks

Comment (petervanderkadecent) [reply src/api/AccountApi.ts]: Done.

**Pull request closed without merging**